controller/controller.go

```diff
@@ -154,6 +159,21 @@ func New(config Config) (*Controller, error) {
 		}
 	}
 
+	var sentryClient *sentry.Service
+	{
+		c := sentry.Config{
+			Dsn: config.SentryDSN,
+		}
+		sentryClient, err = sentry.New(c)
+		if err != nil {
+			// Error during sentry initialization.
+			config.Logger.LogCtx(context.Background(), "level", "error", "message", "Error initializing Sentry client", "stack", microerror.JSON(err))
+			if err != nil {
+				return nil, microerror.Mask(err)
+			}
+		}
```

Comment:
I do not follow this here now. What do we want when the sentry initialization produces an error? We should either log it or return it, but not both, because the returned error is logged already. IMO we should just return the error that got forwarded to us.

Reply:
done